draft-ietf-idr-link-bandwidth.xml

```diff
@@ -147,12 +147,38 @@
       low-order octet of the extended type field for this communities is 0x04.
       The value of the Global Administrator subfield in the Value Field SHOULD
       represent the Autonomous System of the router that attaches the Link
-      Bandwidth Community, but in can be set to any 4B value. If four octet AS
-      numbering scheme is used <xref target="RFC6793"/>, AS_TRANS should be
-      used in the Global Administrator subfield. The bandwidth of the link is
-      expressed as 4 octets in <xref target="IEEE.754-2019"/> floating point
-      format, units being bytes (not bits!) per second. It is carried in the
-      Local Administrator subfield of the Value Field.</t>
+      Bandwidth Community, but in can be set to any 2-byte value. If four
+      octet AS numbering scheme is used <xref target="RFC6793"/>, AS_TRANS
+      should be used in the Global Administrator subfield. The bandwidth of
+      the link is expressed as 4 octets in <xref target="IEEE.754-2019"/>
+      floating point format, units being bytes (not bits!) per second. It is
+      carried in the Local Administrator subfield of the Value Field.</t>
+
+      <figure anchor="LBWExtCom" suppress-title="false"
+              title="Link Bandwidth Extended Community">
+        <artwork align="left" xml:space="preserve">
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|   Type=0x00/0x40   | SubType= 0x04 |       AS Number          |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                    Link Bandwidth Value                       |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+ Type:   1-octet field MUST be set to 0x00 or 0x40 
+         to indicate transitive/non-transitive.
+
+ SubType: 1-octet field MUST be set to 0x04 
+          to indicate 'Link-Bandwidth'.
+
+ Global Administrator sub-field: 
+          2-octet represent the Autonomous System.
+
+ Local Administrator sub-field: 
+          This bandwidth value is encoded as 4 octets 
```

Comment:
Bandwidth value (bytes per sec) encoded as 4 octets
in IEEE floating point format.

Reply:
I have made this change. Please review.